s64da_benchmark_toolkit/prepare.py

```diff
@@ -176,18 +176,30 @@ def _load_pre_schema(self, conn):
                 conn.cursor.execute(pre_schema_file.read())
 
     def _load_license(self, conn):
-        license_path = '/s64da.license'
         try:
+            conn.cursor.execute(f'select swarm64da.show_license()')
+            license_path = self.args.s64da_license_path
             conn.cursor.execute(f'select swarm64da.load_license(\'{license_path}\')')
-            print(f'Loading S64 DA License {license_path}')
-        except:
-            print(f'Could not find a license at {license_path}. Skip loading license.')
 
-        print(f'S64 DA License Status:')
-        try:
-            self._run_shell_task(f'psql {self.args.dsn} -c "select swarm64da.show_license()"')
-        except:
-            print(f'No license file loaded or license invalid.')
+            conn.cursor.execute(f'select swarm64da.show_license()')
+            license_status = conn.cursor.fetchall()[0]
+            print(f'S64 DA license status: {license_status}')
+
+        except errors.InvalidSchemaName as err:
+            print(f'Schema swarm64da not found: {err}')
+            raise
+
+        except errors.UndefinedFunction as err:
```

Comment:
That will crash in AWS because or .so does not have that function compiled for AWS